x-pack/test/stack_functional_integration/apps/monitoring/_monitoring_metricbeat.js

```diff
@@ -4,41 +4,90 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+import expect from '@kbn/expect';
 
 export default ({ getService, getPageObjects }) => {
   describe('monitoring app - stack functional integration - suite', () => {
     const browser = getService('browser');
     const PageObjects = getPageObjects(['security', 'monitoring', 'common']);
-    const log = getService('log');
+    //const log = getService('log');
     const testSubjects = getService('testSubjects');
-    const isSaml = !!process.env.VM.includes('saml') || !!process.env.VM.includes('oidc');
+    //const esClusterSummaryStatus = getService('monitoringElasticsearchSummaryStatus');
     const clusterOverview = getService('monitoringClusterOverview');
     const find = getService('find');
 
     before(async () => {
-      await browser.setWindowSize(1200, 800);
-      if (process.env.SECURITY === 'YES' && !isSaml) {
-        await PageObjects.security.logout();
-        log.debug('### log in as elastic superuser to enable monitoring');
-        // Tests may be running as a non-superuser like `power` but that user
-        // doesn't have the cluster privs to enable monitoring.
-        // On the SAML config, this will fail, but the test recovers on the next
-        // navigate and logs in as the saml user.
-      }
-      // navigateToApp without a username and password will default to the superuser
+      await browser.setWindowSize(1200, 1200);
       await PageObjects.common.navigateToApp('monitoring', { insertTimestamp: false });
       await clusterOverview.acceptAlertsModal();
     });
-
-    it('should have Monitoring already enabled', async () => {
+    beforeEach(async () => {
+      await PageObjects.common.navigateToApp('monitoring', { insertTimestamp: false });
+      //await clusterOverview.acceptAlertsModal();
       await find.clickByLinkText('elasticsearch');
+      // this is a workaround for https://github.com/elastic/kibana/issues/130029
+    });
+    it('Elasticsearch should have >0 documents', async () => {
```

Comment:
We should avoid using chars like `>` in test names because they can become part of failed test screenshot names and aren't valid filenames.  (actually I think they get stripped out, but then it may not make sense).  Maybe just change to `Elasticsearch should have at least 1 document` or `Elasticsearch should have documents`